contracts/src/Gateway.sol

```diff
@@ -617,4 +621,31 @@ contract Gateway is IGateway, IInitializable {
         assets.assetHubCreateAssetFee = config.assetHubCreateAssetFee;
         assets.assetHubReserveTransferFee = config.assetHubReserveTransferFee;
     }
+
+    /// @inheritdoc IGateway
+    function transact(
```

Comment:
I think in this _user-facing_ API, we should rather call this function `sendCall`. This is also consistent with the naming of `sendToken`.

```suggestion
    function sendCall(
```

Reply:
https://github.com/Snowfork/snowbridge/pull/1141/commits/5084432d3b8c46ad002216bc34f0d94c8ca4dc08